src/features/views/components/ViewDataTable/columnTypes/PersonFieldColumnType.tsx

```diff
@@ -0,0 +1,47 @@
+import { GridColDef } from '@mui/x-data-grid-pro';
+
+import { IColumnType } from '.';
+import { PersonFieldViewColumn, ZetkinViewColumn } from '../../types';
+import { EnumChoice } from 'utils/types/zetkin';
+
+type SimpleData = string | number | boolean | null;
+
+export default class PersonFieldColumnType
+  implements IColumnType<ZetkinViewColumn, SimpleData>
+{
+  cellToString(cell: SimpleData): string {
+    if (this.enumChoices) {
+      const choice = this.enumChoices.find((c) => c.key == cell);
+      return choice ? choice.label : '';
+    } else {
+      return cell != null ? cell.toString() : '';
+    }
+  }
+
+  private enumChoices: EnumChoice[] | null = null;
+
+  getColDef(column: PersonFieldViewColumn): Omit<GridColDef, 'field'> {
+    return {
+      filterable: true,
+      valueGetter: (params) => {
+        const cell = params.row[params.field];
+        if (column.config.enum_choices) {
+          this.enumChoices = column.config.enum_choices;
```

Comment:
Setting `this.enumChoices` here and then relying on it in several other places feels risky. This instance is not for one specific column, it's for all columns of this type. So there is nothing guaranteeing (at least not type/syntax-wise) that this method will be called before, say, `cellToString()` or that this method can't be called for another method before calling `cellToString()`.

Most likely this will fail for views that have more than one column of enum type.

Reply:
The code no longer relies on `this.enumChoices`, but the variable is still there and should be removed. I've commented on that separately, so I consider this to be resolved now.

Reply:
This is fixed